src/types/log.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * Log Levels
+ *
+ * Check the main README to see how to access the `LogLevel` enum.
+ */
+export enum LogLevel {
+  Debug = 0,
+  Warning = 1,
+  Error = 2,
+  Trace = 3,
+  Info = 4,
+  Fatal = 5,
+}
+
+/**
+ * @hidden
+ */
+export const LOG_LEVELS: LogLevel[] = [
+  LogLevel.Debug,
+  LogLevel.Warning,
+  LogLevel.Error,
+  LogLevel.Trace,
+  LogLevel.Info,
+  LogLevel.Fatal,
+];
+
+/**
+ * @hidden
+ */
+export interface ILogEntryShort {
+  x: string; // Date ISO
+  t: string; // Text
+  m: string; // Method
+  at: number; // Absolute Time
+  tg: string; // Tag
+  f: string; // File
+  l: number; // Line
+  ll: number; // Level
+  u: string; // URL
```

Comment:
I know you copied this from the Web SDK… but now that I see it, I think is better with `/** … */` as it'll show the name on the IDE. Same as what we do in `ILogEntry`.

```suggestion
  /** Date ISO */
  x: string;
  /** Text */
  t: string;
  /** Method */
  m: string;
  /** Absolute Time */
  at: number;
  /** Tag */
  tg: string;
  /** File */
  f: string;
  /** Line */
  l: number;
  /** Level */
  ll: number;
  /** URL */
  u: string;
```